packages/vuetify/src/components/VBanner/VBanner.sass

```diff
@@ -5,6 +5,9 @@
 +theme('v-banner.v-sheet') using ($material)
   background-color: transparent
 
+  &.v-sheet--outlined
+    background: white !important
```

Comment:
Probably not great on a dark theme, and I doubt `!important` is necessary.

Reply:
how about adding `!important` to `  background-color: transparent` instead of adding white bg ? 
`!important`  will override any color if exist (when adding color to the banner)

Reply:
Note: adding color will add a class for  background with `! important` so ot need to be overridden with another `important` 😁

Reply:
I've added a new commit, updated description